packages/titus-backend/lib/config/index.js

```diff
@@ -56,7 +57,24 @@ module.exports = {
     poolSize: 10,
     idleTimeoutMillis: 30000
   },
-  underPressure: {},
+  underPressure: {
```

Comment:
while we are at it, I would also add some basic config for under-pressure:

```js
{
  maxHeapUsedBytes: 768 * 1024 * 1024, // arbitrary, 768 MB of RAM
  maxRssBytes: 1024 * 1024 * 1024, // arbitrary, 1 GB of RAM
  maxEventLoopUtilization: 0.98
}
```

Reply:
done and pushed, but it's not showing up yet 🤔